api/app/src/external/fhir/patient/index.ts

```diff
@@ -18,29 +18,36 @@ export const toFHIR = (patient: Patient): FHIRPatient => {
     identifier: convertDriversLicenseToIdentifier(patient.data.personalIdentifiers),
     name: [
       {
-        family: patient.data.lastName,
-        given: [patient.data.firstName],
+        family: patient.data.lastName[0],
```

Comment:
Do we need to send it indexed? If `family` is an array, we can just send `lastName`... if `family` is a string, shouldn't we join `lastName`'s items?